src/account.ts

```diff
@@ -16,10 +16,14 @@ const accountFields = {
   code: {
     type: GraphQLString,
     resolve: ({ address }) => web3.eth.getCode(address)
+  },
+  transactionCount: {
+    type: longType,
```

Comment:
I think you might be able to use a standard `GraphQLInt` for a type here – we don't expect the TX count to exceed a 32-bit int.